specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/examples/ServerExtendedAuditingSettingsList.json

```diff
@@ -0,0 +1,31 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "blobauditingtest-4799",
+    "serverName": "blobauditingtest-6440",
+    "api-version": "2017-03-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/blobauditingtest-6852/providers/Microsoft.Sql/servers/blobauditingtest-2080/extendedAuditingSettings/default",
```

Comment:
>default [](start = 187, length = 7)

Can there be more than one? (`default` in other places implies a ARM singleton resource pattern)

Reply:
Cannot be more than one.